backend/src/accounts.ts

```diff
@@ -75,10 +75,10 @@ export const getAccountInfo = async (accountId: string) => {
   return {
     accountId: accountInfo.account_id,
     createdByTransactionHash:
-      accountInfo.created_by_transaction_hash || undefined,
+      accountInfo.created_by_transaction_hash || "Genesis",
```

Comment:
@luixo I just want to share the idea behind `undefined` vs "Genesis"; when the variable named with a clear type in mind ("Hash"), I do not expect to see some arbitrary value there (`Genesis`), which is why I preferred using `undefined`

Reply:
Totally agree with that.
Addressed it here: https://github.com/near/near-explorer/pull/1007